### DISCUSSION TITLE
Is there a way to output rate limiter logs to info?

### DISCUSSION THREAD

**Discussion by DonaldDatThanhNguyen** [emqx/emqx#13004] [Q&A] [answered]
I am currently testing the rate limiter feature for EMQx v5.6.1. 

I noticed that the log is only visible at the debug level:

> 2024-04-29T14:31:14.217190-07:00 [**debug**] clientid: mqttx_a20cb0bd_4, msg: pause_time_due_to_rate_limit, peername: 127.0.0.1:55746, needs: [{32,bytes},{1,messages}], time_in_ms: 200

This is not helpful since we will not be able to turn on debug logs in a production environment.

Is there a way to change the rate limiter to log to the info level?

**Comment by lafirest** [comment]
@DonaldDatThanhNguyen 
Currently, there is no way to configure that.

**Reply by lafirest** [reply, marked as answer]
Update: we are going to lift the log level to `warning` in EMQX 5.8

**Reply by DonaldDatThanhNguyen** [reply]
Do you have ETA when v5.8 will be release? Thanks.

**Reply by lafirest** [reply]
Sorry, I don't know the certain date, but it will be coming soon

**Reply by zmstone** [reply]
https://github.com/emqx/emqx/pull/13009
